README.md

```
@@ -59,10 +59,16 @@ Below is an enhanced explanation for the **Automated Version Calculation** featu
    ```
 2. **Build the project:**
    ```bash
+   # Standard build (uses system OpenSSL)
    cargo build --release
+   
+   # Build with vendored OpenSSL (standalone binary)
+   cargo build --release --features vendored
    ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)